src/data/__snapshots__/redux.test.jsx.snap

```diff
@@ -59,6 +59,7 @@ Object {
       "time_limit_mins": 30,
       "type": "timed",
     },
+    "initialDataLoaded": true,
```

Comment:
`initial` is not clear enough could we rename it to a more accurate name e.g.:  `isLoadingAttemptData`